packages/manager/modules/pci/src/projects/project/storages/databases/components/allowed-ip/allowed-ip.constants.js

```diff
@@ -0,0 +1,5 @@
+export const IPV4_BLOCK_PATTERN = /^(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])(\/([0-9]|[1-2][0-9]|3[0-2]))?$/;
```

Comment:
do you have a source for this one ? RFC ?
maybe add a comment with the source so we know where it comes from